pynance/data/retrieve.py

```diff
@@ -10,24 +10,29 @@
 Wraps `Pandas Remote Data Access 
 <http://pandas.pydata.org/pandas-docs/stable/remote_data.html>`_.
 """
-
+from datetime import datetime
 from ftplib import FTP
 from functools import partial
 import io
 
 import pandas_datareader.data as web
 
-def get(equity, start, end):
+def get(equity, *args, **kwargs):
     """ 
-    Get DataFrame for an individual equity from Yahoo!  
-    
+    get(equity, start = None, end = None)
```

Comment:
The convention is not to have space around '=' in the function definition. 'start=None', not 'start = None'